lib/ws_servers/api/send_authenticated.js

```diff
@@ -49,6 +49,6 @@ module.exports = async (ws, db, d, opts) => {
   notifySuccess(ws, `Decrypted credentials for bitfinex (${mode})`)
   send(ws, ['data.api_credentials.configured', 'bitfinex'])
 
-  ws.aoc.openHost('bitfinex', key, secret)
+  ws.algoWorker.start(key, secret, 'HF_User')
```

Comment:
will this userId vary at some point? I saw that this string constant is being used in more than one place in the code base, maybe we could centralize it WDYT?

Reply:
not for the desktop app

Reply:
will do!

Reply:
fixed